### PR TITLE
Basic styling

### PR DESCRIPTION
**Pull request by ahohnsen**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,33 +4,55 @@
...
+      <section class="section-box" id="interests"><h2>Interests</h2>
+        <ul>
+         <li>Interests 1</li>
+         <li>Interests 2</li>
+         <li>Interests 3</li>
+         <li>Interests 4</li>
+         <li>Interests 5</li>
+        </ul>
+       </section></div>
+       <section id="follow">
```
Comment (JelenadlS) [inline index.html:42]: I cannot find the #follow above so that is not necessary. But i also might oversaw it or wrong :-)

File: index.html [hunk 2]
```diff
@@ -7,31 +7,52 @@
...
+        <p><b>Follow me on:</b></p>
```
Comment (FaGru) [inline index.html:43]: looks all good 👍🏻 
maybe i only would take a `h2` at follow me, but i'm not sure 🤷🏻

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+  
+  .button {
+    display: flex;
+    justify-content: center;
+    color: black;
+    text-decoration: none;
+    padding: 10px;
+  }
+  
+  .button:hover {
```
Comment (JelenadlS) [inline styles.css:38]: nice idea :-)

**Review by JelenadlS — APPROVED** [review]
I think it looks very good and well structured! 🥳

File: index.html (hunk 1 above)
Comment (ahohnsen) [reply index.html:42]: You are right! Thank you, I will correct it.

File: index.html (hunk 2 above)
Comment (ahohnsen) [reply index.html:43]: Thanks, I just left it like this for the moment. Still need to work on that section.

**Pull request merged**